valkey.conf

```diff
@@ -1841,6 +1841,38 @@ aof-timestamp-enabled no
 #
 # cluster-replica-no-failover no
 
+# The timeout in milliseconds for cluster manual failover. If a manual failover
+# does not complete within the specified time, both the replica and the primary
+# will abort it.
+#
+# A manual failover is a special kind of failover that is usually executed when
+# there are no actual failures, but we wish to swap the current primary with one
+# of its replicas (which is the node we send the CLUSTER FAILOVER command to) in
+# a safe way, without any window for data loss.
+#
+# It works in the following way:
+# 1. The replica tells the primary to stop processing queries from clients.
+# 2. The primary replies to the replica with the current replication offset.
+# 3. The replica waits for the replication offset to match on its side, to make
+#    sure it processed all the data from the primary before it continues.
+# 4. The replica starts a failover, obtains a new configuration epoch from the
+#    majority of the masters, and broadcasts the new configuration.
+# 5. The old primary receives the configuration update: unpause its clients and
+#    starts replying with redirection messages so that they'll continue the chat
+#    with the new primary.
+#
+# This way clients are moved away from the old primary to the new primary atomically
+# and only when the replica that is turning into the new primary has processed
+# all of the replication stream from the old primary.
```

Comment:
i took this from CLUSTER FAILOVER command docs, let me know if it is too much, please also feel free to insert a suggestion if you have a good wording.

Reply:
Could we also mention what's the behavior if it's set to 0, is it indefinite timeout?

Reply:
i think we should set a min and max value of it (that is not allow 0), but i am not sure about the value, do you have some ideas?

Reply:
I agree, I don't think a value of 0 makes sense since there is no way to abort (afaik, you might be able to trigger another failover). I would vote a minimum time of like 1000ms.

Reply:
do we also want a max value? The big value also does not makes senese and we will hit the overflow issue when doing this `now + (server.cluster_mf_timeout * CLUSTER_MF_PAUSE_MULT)` if someone test it.

Reply:
I don't have a strong opinion, but agree a high value is bad since it's not interruptible. Maybe like a minute? (Seems like such a small range, maybe minimum should be lower, like 250ms)

Reply:
[250, 60000] SGTM, i changed to it.